pgadmin/templates/deployment.yaml

```
@@ -0,0 +1,62 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: pgadmin
+  labels:
+    app: pgadmin
```

Add the labels from helpers.tpl 
```suggestion
{{ include "pgadmin.labels.stakater" . | indent 4 }}
{{ include "pgadmin.labels.chart" . | indent 4 }}
```